sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql

```diff
@@ -1,4 +1,5 @@
 SELECT
+  DATE(submission_timestamp) AS submission_date,
```

Comment:
this change is not strictly needed to fix the Airflow error, right? (just checking for my own understanding)

Reply:
Apologies for shoehorning this in here. The changes in this file are indeed for a [separate error](https://workflow.telemetry.mozilla.org/log?task_id=search_terms_derived__adm_weekly_aggregates__v1&dag_id=bqetl_search_terms_daily&execution_date=2021-09-26T03%3A00%3A00%2B00%3A00) that's similar.